src/apps/api/tests/test_competition_participants.py

```diff
@@ -46,3 +48,13 @@ def test_normal_user_cannot_see_participants(self):
         resp = self.get_participants()
         assert CompetitionParticipant.objects.filter(competition=self.comp).count() > 0
         assert len(resp.json()) == 0
+
+    def test_cant_participate_multiple_times(self):
+        self.client.login(username='norm', password='norm')
+        resp = self.client.post(reverse('competition-register', kwargs={"pk": self.comp.pk}))
+        assert resp.status_code == 201
+        CompetitionParticipant.objects.get(user=self.norm)
```

Comment:
I assume this is verifying that the CompetitionParticipant was created? probably better to do something like 
```suggestion
        assert CompetitionParticipant.objects.filter(user=self.norm, competition=self.comp).exists()
```
so we don't get a `CompetitionParticipant.DoesNotExist` error in tests

Reply:
✔️